SKILL.tsv

```diff
@@ -2585,7 +2585,7 @@ SKILL_20150317_002584	{memo X}$Increases the physical attack of [Kneeling Shot]
 SKILL_20150317_002585	$Steady Aim: Enhance
 SKILL_20150317_002586	{memo X}$Increases the additional damage of [Steady Aim] per attribute level.
 SKILL_20150317_002587	$Sacrament: Enhance
-SKILL_20150317_002588	$Increases the damage on Holy property of [Sacrament] per attribute level.
+SKILL_20150317_002588	$Increases the Holy property damage of [Sacrament] per attribute level.
```

Comment:
Per attribute level does not fit here. Only should be used if a set amount is denoted.